<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWindow</class>
 <widget class="QDockWidget" name="HistoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>History</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>520</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="historyListWidget_"/>
     </item>
     <item>
      <widget class="QTextEdit" name="networkXMLTextEdit_">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="undoAllButton_">
       <property name="text">
        <string>Undo All</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="undoButton_">
       <property name="text">
        <string>Undo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="redoButton_">
       <property name="text">
        <string>Redo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="redoAllButton_">
       <property name="text">
        <string>Redo All</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
